azure-spring-apps-maven-plugin/src/main/java/com/microsoft/azure/maven/springcloud/config/AppRawConfig.java

```diff
@@ -5,14 +5,32 @@
 
 package com.microsoft.azure.maven.springcloud.config;
 
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudApp;
 import lombok.Data;
 
+import javax.annotation.Nonnull;
+
 @Data
 public class AppRawConfig {
     private String subscriptionId;
     private String resourceGroup; // optional
     private String clusterName;
     private String appName;
     private String isPublic;
+    private String region;
+    private String sku;
+    private String environment;
+    private String environmentResourceGroup;
```

Comment:
how about adding a `ClusterRawConfig`, as an alternative config to `clusterName`.